aggregator/src/app/helpers/plus.ts

```diff
@@ -0,0 +1,3 @@
+export default function plus(a: number, b: number) {
+  return a + b;
+}
```

Comment:
Must've missed this last round, seems like overkill to have this separated out when something like (acc: number, numActions: number) => acc + numActions` would do and be more descriptive.

Reply:
@jacque006 Hmm. As a reducer function I think `plus` is a bit simpler than a lambda that tries to add more context. Also, `plus` is used in two places so using lambdas would be a bit more verbose.

Reply:
Ah I missed the second usage, makes a little more sense now.